### DISCUSSION TITLE
Trouble relaying an opus stream to another server

### DISCUSSION THREAD

**Discussion by Flopblopper** [savonet/liquidsoap#2909] [Q&A]
Hello,

I'm trying to create a Liquidsoap script that will take a stream from a local Icecast server, and send it to a remote server. I've got it working for the most part, but it breaks whenever the track changes, and cannot be started again until I restart Icecast. When it stops working, Liquidsoap displays an error stating that it could not identify the stream.

I was wondering if someone could take a look at my script and let me know if you see any obvious errors.

```
source = input.http("http://127.0.0.1:8000/Stream.opus")
output.icecast(%ffmpeg(format="opus", %audio.copy), fallible=true, mount="/Stream.opus", host="fakesite.com", port=8000, user="source", password="hackme", source)
```

Thanks

**Comment by toots** [comment]
Hi, do you have some logs?

Also, I would suggest doing the same as in https://github.com/savonet/liquidsoap/issues/2917#issuecomment-1442081440 and bumping the ogg decoder priority:
```
settings.decoder.priorities.ogg.set(20)
```
